internal/plugins/golang/v2/api.go

```diff
@@ -46,15 +47,11 @@ func (p *createAPISubcommand) Run() error {
 	for _, r := range p.config.Resources {
 		oldResources[r] = struct{}{}
 	}
+
 	if err := p.CreateAPISubcommand.Run(); err != nil {
 		return err
 	}
 
-	// Emulate plugins phase 2 behavior by checking the config for this plugin's config object.
-	if !hasPluginConfig(p.config) {
-		return nil
-	}
-
```

Comment:
Why we remove it? if is project-version 2 should not add the SDK plugins on top. So, by removing are we not introducing bug?